docs/how-to-renew-docker-mirror-proxy-ca-cert.md

```diff
@@ -0,0 +1,24 @@
+# Renewing the CA Certificate for the docker-mirror-proxy
+
+Prow jobs will fail to pull container images if the docker-mirror-proxy CA certificate is allowed to expire. 
+
+There is a [periodic job](https://github.com/kubevirt/project-infra/blob/dcd4fd00a2c1592e0db577bd824314cdd4221818/github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-periodics.yaml#L541) in place to check that the CA certificate will not expire within the next 90 days. 
```

Comment:
nit: add a link to the script referring to this doc page

Reply:
I added the document to the script.